### ISSUE TITLE
portal bug in example-test

### ISSUE THREAD

**Issue by Abirdcfly** [kubebb/core#274]
```
7.2 create another portal with the same path and entry
portal.core.kubebb.k8s.com.cn/portal-example-duplicate created
Timeout reached
```
see https://github.com/kubebb/core/actions/runs/5920999107/job/16053036729?pr=248#step:6:418 and
https://github.com/kubebb/core/actions/runs/5921044842/job/16053269895#step:6:418

kind log https://github.com/kubebb/core/suites/15290484579/artifacts/873115567

**Comment by Abirdcfly** [comment]
by the way update with https://github.com/kubebb/core/pull/259#discussion_r1298258203

**Comment by bjwswang** [comment]
@Abirdcfly  I will fix this along with #273
